### PR TITLE
Software update docs

### PR DESCRIPTION
**Pull request by vathzen**
Added changes to helmi-qiskit and helmi-cirq notebook to reflect qiskit-1.0 changes.
Visualize Quality Metrics notebook changes are a WIP

### CODE REVIEW

File: docs/lumi.md [hunk 1]
```diff
@@ -45,15 +45,15 @@ Loading `helmi_qiskit` or `helmi_cirq` on LUMI comes with a preconfigured Python
...
+| [iqm-client ](https://pypi.org/project/iqm-client/) | >=18.0 < 19.0  |
+| [qiskit-iqm](https://pypi.org/project/qiskit-iqm/)  | >=13.15 < 14.0 |
+| [cirq-iqm](https://pypi.org/project/qiskit-iqm/)    | >=14.4 < 15.0  |
 
 </center>
 
 Newer versions of the above Python packages can be installed and may work with Helmi, however these are currently unsupported and may lead to errors.
 
-As an alternative to the above, Python packages can be installed on top of `helmi_qiskit` or `helmi_cirq` into the Python user install directory by specifying `python -m pip install --user whatsapp`. This, however, may lead to increased dependency conflicts.
+As an alternative to the above, Python packages can be installed on top of `helmi_qiskit` or `helmi_cirq` into the Python user install directory by specifying `python -m pip install --user <username>`. This, however, may lead to increased dependency conflicts.
```
Comment (JMuff22) [inline docs/lumi.md]: Please revert the change. The `--user` flag does not take in argument username

File: docs/examples/intro-to-helmi-qiskit.ipynb [hunk 2]
```diff
@@ -575,9 +731,25 @@
...
-   "outputs": [],
+   "execution_count": 52,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/sssriva/miniconda3/envs/qc-docs/lib/python3.11/site-packages/iqm/qiskit_iqm/iqm_provider.py:159: UserWarning: Unknown backend option(s): {'seed_simulator'}\n",
+      "  warnings.warn(f'Unknown backend option(s): {unknown_options}')\n"
```
Comment (JMuff22) [inline docs/examples/intro-to-helmi-qiskit.ipynb]: Please fix.

File: docs/lumi.md (hunk 1 above)
Comment (vathzen) [reply docs/lumi.md]: What is the significance of the "whatsapp" then?

Comment (JMuff22) [reply docs/lumi.md]: It's an example python package. For example it could be `python -m pip install --user <python-package>`

**Comment by vathzen** [comment]
@JMuff22 pre-commit.ci failing due to spell checks in image encoding.

**Comment by JMuff22** [comment]
> @JMuff22 pre-commit.ci failing due to spell checks in image encoding. 

Ok @vathzen lets remove the pre-commit spell check in the configuration. It worked for a while but now no longer seems to be the case

File: docs/lumi.md [hunk 3]
```diff
@@ -45,9 +45,9 @@ Loading `helmi_qiskit` or `helmi_cirq` on LUMI comes with a preconfigured Python
 
 | Package                                             | Version        |
 | --------------------------------------------------- | -------------- |
-| [iqm-client ](https://pypi.org/project/iqm-client/) | >=15.2 < 16.0  |
-| [qiskit-iqm](https://pypi.org/project/qiskit-iqm/)  | >=11.10 < 12.0 |
-| [cirq-iqm](https://pypi.org/project/qiskit-iqm/)    | >=12.2 < 13.0  |
+| [iqm-client ](https://pypi.org/project/iqm-client/) | >=18.0 < 19.0  |
+| [qiskit-iqm](https://pypi.org/project/qiskit-iqm/)  | >=13.15 < 14.0 |
```
Comment (JLenssen) [inline docs/lumi.md]: Those should be updated to the latest versions

Comment (vathzen) [reply docs/lumi.md]: fixed in latest commit

File: docs/calibration.md [hunk 4]
```diff
@@ -1,22 +1,12 @@
...
-- Adjusts qubit drive frequency, drive amplitude and readout threshold
-- Measures $T_1$, $T_2$, $T_2^*$ and readout accuracy
+To minimize the impact of calibration on user operations, we calibrate Helmi at night, by running a specific sequence of experiments in order.
 
-**Extended calibration**:
+**Calibration**:
 
-- Every day at 4 am
+- Every day at 2 am
 - Adjusts qubit drive frequency, amplitude fine-tuning and readout threshold
```
Comment (JLenssen) [inline docs/calibration.md:10]: 2QB gate calibration is missing

File: docs/calibration.md [hunk 5]
```diff
@@ -49,20 +39,27 @@ Here is an example response from Helmi's API for the calibration and quality met
...
+  "calibration_set_end_timestamp": "2024-11-26 03:03:10.542733+00:00",
+  "calibration_set_is_invalid": false,
+  "quality_metric_set_id": "e9d1c497-3485-4d9b-b176-2c0b100a5ccd",
+  "quality_metric_set_created_timestamp": "2024-11-26 03:03:10.496040+00:00",
+  "quality_metric_set_end_timestamp": "2024-11-26 03:03:10.569614+00:00",
+  "quality_metric_set_is_invalid": false,
   "metrics": {
-    "QB1.fidelity_1qb_gates_averaged": {
-      "value": "0.9971526901996984",
+    "metrics.ssro.measure.constant.QB1.fidelity": {
```
Comment (JLenssen) [inline docs/calibration.md]: This is the IQM format, here we should show our own format.

File: docs/calibration.md (hunk 4 above)
Comment (JLenssen) [reply docs/calibration.md:10]: In the section below, we should explain the `par=d2` parameter

File: docs/limitations.md [hunk 6]
```diff
@@ -18,8 +18,8 @@ ClientAuthenticationError: Authentication failed: {"detail":"Job rejected: Too m
 
 **Batch size**
 
-On our Helmi system, quantum circuits within a batch are processed sequentially. To prevent extensive queue occupation by large batches, we have set a maximum limit of **20 circuits per batch**.
-Jobs submitting batches with more than 20 circuits will be rejected and the IQM client will return the following error
+On our Helmi system, quantum circuits within a batch are processed sequentially. To prevent extensive queue occupation by large batches, we have set a maximum limit of **200 circuits per batch**.
+Jobs submitting batches with more than 200 circuits will be rejected and the IQM client will return the following error
 
 ```bash
 ClientAuthenticationError: Authentication failed: {"detail":"Too many circuits X in batch (max: 20)"}
```
Comment (JLenssen) [inline docs/limitations.md]: Is this still the correct error message?

**Comment by JLenssen** [comment]
Can we also double check the CZ gate length for Helmi and udpate the value if needed?

**Comment by JLenssen** [comment]
Did you test heralding on Helmi? I believe it works but should be double checked.

File: docs/calibration.md [hunk 7]
```diff
@@ -1,22 +1,12 @@
 Helmi is continuously calibrated to ensure functionality. The calibration process involves a series of experiments, aimed at fine-tuning the parameters, necessary for controlling and measuring the qubits. In addition to calibration, we run benchmarks to obtain the figures of merit reflecting the current state of the quantum computer.
 
-!!! note
-
-    Calibration and benchmarking experiments are interleaved with regular user jobs in Helmi's job queue, as detailed in the [Running on Helmi](running.md) section. The calibration might therefore slightly increase the waiting time in the queue of regular user jobs.
-
```
Comment (JMuff22) [inline docs/calibration.md:6]: I believe this is still relevant. Even if the calibration runs at 2am, users may still experience increased queue times if they run over night for example.

File: docs/examples/intro-to-helmi-cirq.ipynb [hunk 8]
```diff
@@ -86,15 +97,18 @@
...
+    "\n",
     "sampler = IQMSampler(\n",
-    "    \"https://qc.vtt.fi/helmi/cocos\",\n",
-    "    calibration_set_id=\"2318283e-3adf-4a88-a936-a33affde4af7\",\n",
-    "    max_circuit_duration_over_t2=1.0,\n",
-    "    heralding_mode=\"zeros\",\n",
+    "    os.environ[\"HELMI_CORTEX_URL\"],\n",
+    "    calibration_set_id=\"e0cf7953-1eb8-4c71-a262-301380baa2c4\",\n",
+    "    compiler_options=CircuitCompilationOptions(\n",
+    "        max_circuit_duration_over_t2=None, heralding_mode=\"zeros\"\n",
```
Comment (JMuff22) [inline docs/examples/intro-to-helmi-cirq.ipynb]: Maybe use the syntax from IQM docs 

```
from iqm.iqm_client import CircuitCompilationOptions

job = backend.run(transpiled_circuit, shots=1000, circuit_compilation_options=CircuitCompilationOptions(heralding_mode=HeraldingMode.ZEROS))
```

**Review by JMuff22 — CHANGES_REQUESTED** [review]
Looks good, small changes. I would also like to highlight in the qiskit notebook instantiating the backend with a calibration set id and if you do not specify a calibration set id then it uses the default. e.g from the IQM docs 

> The calibration data for an IQM quantum computer is stored in a calibration set. An [IQMBackend](https://iqm-finland.github.io/qiskit-on-iqm/api/iqm.qiskit_iqm.iqm_provider.IQMBackend.html#iqm.qiskit_iqm.iqm_provider.IQMBackend) instance always corresponds to a specific calibration set, so that its transpilation target uses only those QPU components (qubits and computational resonators) and gates which are available in that calibration set. The server default calibration set will be used by default, but you can also use a different calibration set by specifying the calibration_set_id parameter for [IQMProvider.get_backend()](https://iqm-finland.github.io/qiskit-on-iqm/api/iqm.qiskit_iqm.iqm_provider.IQMProvider.html#iqm.qiskit_iqm.iqm_provider.IQMProvider.get_backend)

**Comment by JMuff22** [comment]
Additional files that require changes:
1. `calibration.md` for example reviewing the calibration metrics and quality metrics section. And double checking the API paths. For example a note on the format of calibration metrics and what `par=d2` means for example. And updated the example calibration response. 
2. Limitations and errors need to be checked and updated 
3. I would like to review `running.md`and check if those instructions still work with LUMI right now.

File: docs/examples/intro-to-helmi-cirq.ipynb
Comment (JLenssen) [inline docs/examples/intro-to-helmi-cirq.ipynb:1]: This one links to https://github.com/iqm-finland/qiskit-on-iqm/tree/9.0 the version should be updated.

File: docs/examples/visualize_quality_metrics.ipynb [hunk 9]
```diff
@@ -114,9 +113,17 @@
...
...mplementation': 'xy_cz', 'parameter': 'par=d2'}, 'QB3__QB4': {'value': '0.9439582461347993', 'unit': '', 'uncertainty': '0.00119987380040661', 'timestamp': '2024-11-19 04:55:27.396209+00:00', 'implementation': 'xy_cz', 'parameter': 'par=d2'}, 'statistics': {'average': 0.9372832243355633, 'median': 0.9488898006623872}}, 'cz_irb_fidelity': {'QB1__QB3': {'value': '0.9855008776293077', 'unit': '', 'uncertainty': '0.0015238326332719856', 'timestamp': '2024-11-19 04:55:27.396209+00:00', 'implementation': 'tgss', 'parameter': 'par=d2'}, 'QB3__QB5': {'value': '0.9854688635058743', 'unit': '', 'uncertainty': '0.0014956904073850882', 'timestamp': '2024-11-19 04:55:27.396209+00:00', 'implementation': 'tgss', 'parameter': 'par=d2'}, 'QB2__QB3': {'value': '0.9439826241533328', 'unit': '', 'uncertainty': '0.003954673659719742', 'timestamp': '2024-11-19 04:55:27.396209+00:00', 'implementation': 'tgss', 'parameter': 'par=d2'}, 'QB3__QB4': {'value': '0.9805787195442951', 'unit': '', 'uncertainty': '0.002085397728623927', 'timestamp': '2024-11-19 04:55:27.396209+00:00', 'implementation': 'tgss', 'parameter': 'par=d2'}, 'statistics': {'average': 0.9738827712082024, 'median': 0.9830237915250847}}}}\n"
```
Comment (JLenssen) [inline docs/examples/visualize_quality_metrics.ipynb]: I would prefer that we clear the output before merging

File: docs/examples/visualize_quality_metrics.ipynb [hunk 10]
```diff
@@ -203,14 +449,18 @@
...
+    "                if qb.startswith(\"QB\"):\n",
+    "                    values.append(float(qb_data[\"value\"]))\n",
+    "                    # Extract the qubit label from the key\n",
+    "                    labels.append(qb)\n",
     "\n",
     "    # Check if values were found for the given metric\n",
     "    if not values:\n",
     "        return f\"{metric} not in quality metrics set!\"\n",
     "\n",
+    "    print(values)\n",
```
Comment (JLenssen) [inline docs/examples/visualize_quality_metrics.ipynb]: Debug leftover?

File: docs/examples/visualize_quality_metrics.ipynb
Comment (JLenssen) [inline docs/examples/visualize_quality_metrics.ipynb:1]: Can we rename to `visualize-quality-metrics.ipynb`?

File: docs/examples/visualize_quality_metrics.ipynb (hunk 9 above)
Comment (JMuff22) [reply docs/examples/visualize_quality_metrics.ipynb]: I agree

File: docs/examples/intro-to-helmi-qiskit.ipynb [hunk 11]
```diff
@@ -142,8 +144,7 @@
    "metadata": {},
    "source": [
     "### Executing the circuit on Helmi\n",
-    "\n",
-    "When submitting a job to Helmi a unique identifier for your job is returned. This can be used to gather additional information about the circuit you just submitted and the results. **You should save your job ids!**"
+    "We now transpile the circuit to rewrite the circuit based on the Helmi topology. Earlier, the qiskit execute automatically transpiled the circuits before running in the backend, but it has since been deprecated in qiskit 1.0+. We can even view the transpiled circuit before running the circuit now as shown: "
```
Comment (JMuff22) [inline docs/examples/intro-to-helmi-qiskit.ipynb]:
```suggestion
    "We now transpile the circuit to run on Helmi. This will rewrite the circuit to take into account Helmi's topology and native gate set. We can also view the transpiled circuit before running the it: "
```

File: docs/examples/intro-to-helmi-qiskit.ipynb [hunk 12]
```diff
@@ -152,17 +153,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "job = execute(circuit, backend, shots=100)\n",
-    "print(f\"Job ID: {job.job_id()}.\")\n",
-    "print(\"Tracking execution of job:\")\n",
-    "job_monitor(job)"
+    "new_circuit = transpile(circuit, backend)\n",
```
Comment (JMuff22) [inline docs/examples/intro-to-helmi-qiskit.ipynb]:
```suggestion
    "transpiled_circuit = transpile(circuit, backend)\n",
```

File: docs/examples/intro-to-helmi-qiskit.ipynb [hunk 13]
```diff
@@ -152,17 +153,15 @@
...
+    "new_circuit.draw(output=\"mpl\")"
```
Comment (JMuff22) [inline docs/examples/intro-to-helmi-qiskit.ipynb]:
```suggestion
    "transpiled_circuit.draw(output=\"mpl\")"
```

File: docs/examples/intro-to-helmi-qiskit.ipynb [hunk 14]
```diff
@@ -171,15 +170,18 @@
...
-    "    for qc in circuit:\n",
-    "        print(qc.draw())\n",
-    "\n",
-    "\n",
-    "job = execute(circuit, backend, shots=100, circuit_callback=print_circ)\n",
-    "print(f\"Job ID: {job.job_id()}.\")\n",
-    "print(\"Tracking execution of job:\")\n",
-    "job_monitor(job)"
+    "backend.set_options(shots=100)\n",
+    "job = backend.run(new_circuit)"
```
Comment (JMuff22) [inline docs/examples/intro-to-helmi-qiskit.ipynb]:
```suggestion
    "job = backend.run(transpiled_circuit)"
```

File: docs/examples/intro-to-helmi-qiskit.ipynb [hunk 15]
```diff
@@ -212,21 +214,34 @@
     "You can also specify some backend arguments for running on Helmi. These are explained in the [qiskit-on-iqm documentation](https://iqm-finland.github.io/qiskit-on-iqm/user_guide.html#installation)."
```
Comment (JMuff22) [inline docs/examples/intro-to-helmi-qiskit.ipynb]:
```suggestion
    "You can also specify some backend arguments for running on Helmi. These are explained in the [qiskit-on-iqm documentation](https://iqm-finland.github.io/qiskit-on-iqm/user_guide.html#executing-a-circuit)."
```

File: docs/examples/intro-to-helmi-qiskit.ipynb [hunk 16]
```diff
@@ -212,21 +214,34 @@
...
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The IQM Provider always corresponds to a `calibration set`, so that while transpilation it uses only those qubits and resonators. The server default calibration set will be used unless one is provided while instantiation."
```
Comment (JMuff22) [inline docs/examples/intro-to-helmi-qiskit.ipynb]:
```suggestion
    "The IQM backend always corresponds to a `calibration set`, so that transpiling using this backend will only use loci defined in the `calibration_set_id`. The server default calibration set will be used unless one is provided while instantiating the backend"
```

File: docs/examples/intro-to-helmi-qiskit.ipynb [hunk 17]
```diff
@@ -212,21 +214,34 @@
...
-    "#     calibration_set_id=\"c5a5e2d1-100b-42d4-a4fd-a49edfb9d176\",\n",
-    "#     max_circuit_duration_over_t2=1.0,\n",
-    "#     heralding_mode=\"zeros\",\n",
-    "#     circuit_callback=None,\n",
-    "# )"
+    "from iqm.iqm_client import CircuitCompilationOptions\n",
+    "\n",
+    "backend = provider.get_backend(calibration_set_id=\"35fa13d1-17db-4bd2-9b14-203ce32f0551\")\n",
+    "\n",
+    "backend.set_options(\n",
```
Comment (JMuff22) [inline docs/examples/intro-to-helmi-qiskit.ipynb]: I think the options should be set during `backend.run`and not `backend.options` . Additionally, the heralding mode enum should be imported from IQMClient. We should explain where users can view the circuit compilations options e.g https://iqm-finland.github.io/iqm-client/api/iqm.iqm_client.models.CircuitCompilationOptions.html#iqm.iqm_client.models.CircuitCompilationOptions . We should also add that Helmi does not contain any Move gates. This can be explained e.g in `helmi.md` where we define the native gates.

File: docs/examples/intro-to-helmi-qiskit.ipynb [hunk 18]
```diff
@@ -212,21 +214,34 @@
...
+    "    shots=100,\n",
+    "    circuit_compilation_options=CircuitCompilationOptions(\n",
+    "        max_circuit_duration_over_t2=None, heralding_mode=\"zeros\"\n",
+    "    ),\n",
+    ")\n",
+    "\n",
+    "new_circuit = transpile(circuit, backend)\n",
+    "opt_circuit = optimize_single_qubit_gates(new_circuit)\n",
```
Comment (JMuff22) [inline docs/examples/intro-to-helmi-qiskit.ipynb]: Optimize single qubit gates should be removed here. Later on in the notebook under "Optimizing circuits for Helmi" we use this.

File: docs/examples/intro-to-helmi-qiskit.ipynb [hunk 19]
```diff
@@ -292,15 +312,14 @@
     "circuit.cx(qreg[0], qreg[1])\n",
     "circuit.measure_all()\n",
     "\n",
-    "\n",
     "# Qubit numbers start at 0 index whereas the qubit names start at 1 index.\n",
     "qubit_mapping = {\n",
     "    qreg[0]: 1,  # Map the first qubit to QB1\n",
```
Comment (JMuff22) [inline docs/examples/intro-to-helmi-qiskit.ipynb]: Actually I think best practice is now to use https://iqm-finland.github.io/qiskit-on-iqm/api/iqm.qiskit_iqm.iqm_backend.IQMBackendBase.html#iqm.qiskit_iqm.iqm_backend.IQMBackendBase.qubit_name_to_index for layout mapping

File: docs/calibration.md [hunk 20]
```diff
@@ -1,27 +1,20 @@
...
 ### Calibration sequences
 
-To minimize the impact of calibration on user operations, we execute shorter calibration sequences during the day and a longer calibration run throughout the night.
-
-**Short calibration**:
+To minimize the impact of calibration on user operations, we calibrate Helmi at night, by running a specific sequence of experiments in order. Calibration and benchmarking experiments are interleaved with regular user jobs in Helmi's job queue, as detailed in the [Running on Helmi](running.md) section. The calibration might therefore slightly increase the waiting time in the queue of regular user jobs.
```
Comment (JMuff22) [inline docs/calibration.md]: "at night". What time specifically?

File: docs/examples/intro-to-helmi-cirq.ipynb [hunk 21]
```diff
@@ -31,8 +31,9 @@
     "This notebook uses the following requirements. \n",
     "\n",
     "```text\n",
-    "cirq-iqm==14.0\n",
-    "iqm-client==17.5\n",
+    "cirq-iqm==15.2\n",
```
Comment (JMuff22) [inline docs/examples/intro-to-helmi-cirq.ipynb]: LUMI currently uses cirq-iqm 15.1

File: docs/running.md [hunk 22]
```diff
@@ -1,9 +1,14 @@
 # Running on Helmi
 
-To run interactive jobs on LUMI the [`srun`](https://slurm.schedmd.com/srun.html) command can be used
+To run interactive jobs on LUMI the [`srun`](https://slurm.schedmd.com/srun.html) command can be used. First we allocate resources for the job:
 
 ```bash
-srun --account=project_<id> -t 0:02:00 -c 1 -n 1 --partition q_industry python qiskit_flip.py
+salloc -n 1 -c 1 -t 00:15:00 --job-name=<job_name> --account=project_<id> --partition q_industry
```
Comment (JMuff22) [inline docs/running.md:6]: We should follow the syntax of the official LUMI docs https://docs.lumi-supercomputer.eu/runjobs/scheduled-jobs/slurm-quickstart/ and therefore, not use `salloc`.`srun` is fine

File: docs/helmi.md [hunk 23]
```diff
@@ -39,7 +39,13 @@ The coupler characteristics are given below. TC refers to tunable coupler.
 | --------------------------------- | ------ | ------ | ------ | ------ |
 | Two-qubit Gate Fidelities[^3] [%] | 96.4   | 95.9   | 94.6   | 97.5   |
 
-The PRX gate length and the CZ gate length is 120 [ns] respectively.
+The CZ gate length are:
```
Comment (JMuff22) [inline docs/helmi.md]: A note should be added that the gate lengths may vary and how can a user query them

File: docs/faq.md [hunk 24]
```diff
@@ -20,20 +20,20 @@ No circuit optimization is performed after the circuit has been submitted in ser
...
+An overhead of 6 seconds can be expected. The largest overhead is introduced by compiling the pulses to instructions for the control electronics and the upload to the instruments.
 
 **Q: Does the number of shots affect the calibration of Helmi?**
 
 No, a large number of shots will not cause thermal excitations of the qubits.
 
 **Q: Can Pennylane be used on Helmi?**
 
-Yes, Pennylane can be used on Helmi through a [fork of the Pennylane-Qiskit package](https://github.com/NordIQuEst/pennylane-qiskit/tree/support-num-qubits). This is demonstrated in the [Introduction to Helmi with Qiskit example](examples/intro-to-helmi-qiskit.ipynb#pennylane-qiskit).
+No, Pennylane is not currently supported on Helmi.
```
Comment (JMuff22) [inline docs/faq.md]: Pennylane is not currently supported on Helmi but pennylane-qiskit may or may not work.

File: docs/faq.md [hunk 25]
```diff
@@ -20,20 +20,20 @@ No circuit optimization is performed after the circuit has been submitted in ser
...
 
 **Q: Is Qiskit v1.0 supported on Helmi?**
 
-Qiskit 1.0 is not currently supported on Helmi. Please refer to the [v0.45.3 API documentation](https://docs.quantum.ibm.com/api/qiskit/0.45).
+Qiskit 1.0 is now supported on Helmi. Please refer to the [v1.1.2 API documentation](https://docs.quantum.ibm.com/api/qiskit/1.1).
```
Comment (JMuff22) [inline docs/faq.md]: We should specify which qiskit version inferring from Qiskit IQM.

**Review by JMuff22 — CHANGES_REQUESTED** [review]
Additionally, the jupyter notebook "Advanced Helmi usage" needs updating and testing. it currently uses old Qiskit syntax. 

I would also add an example of how to do mid circuit measurements in the example notebooks and add an FAQ entry that mid circuit measurements are supported on Helmi but your mileage may vary as this is a new feature

Please remove all notebook outputs before committing.

File: docs/helmi.md [hunk 26]
```diff
@@ -16,8 +16,8 @@ To run arbitrary quantum circuits, a QPU needs to implement a universal gate set
 - Single-qubit gates: Phased-$RX$ gate (PRX)
 - Two-qubit gate: Controlled-$Z$ gate (CZ)
 
-Measurements are natively implemented as single-qubit measurements in the $Z$ basis.
-
+MOVE Gates such as SWAP are not natively implemented on Helmi.
```
Comment (JMuff22) [inline docs/helmi.md]: Remove the reference to SWAP and it's fine

**Comment by vathzen** [comment]
@JMuff22 made the change you suggested, please approve

**Review by JMuff22 — APPROVED** [review]
